src/Properties.cpp

```diff
@@ -12,7 +12,7 @@
 #include "Properties.h"
 
 // ********** GENERAL ********** //
-boolean Properties::DEBUGGING_ACTIVE = false;
+boolean Properties::DEBUGGING_ACTIVE = true;
```

Comment:
Based on my last commit when I flipped this to false, I think we want to keep it that way. We want to assume that we don't want the noise and only turn it on when we do. Lets set this back to false.